src/sentry/api/serializers/models/organization.py

```diff
@@ -238,6 +240,8 @@ def serialize(
         if "server-side-sampling" not in feature_list and "mep-rollout-flag" in feature_list:
             feature_list.remove("mep-rollout-flag")
 
+        has_auth_provider = AuthProvider.objects.filter(organization=obj).exists()
```

Comment:
This is going to do N queries when an organization list is serialized. Could we bulk fetch this in `get_attrs()`

Reply:
@markstory Good point. I've added the bulk fetching to `get_attrs()`.